### PR TITLE
Contact project

### PR DESCRIPTION
**Pull request by yegonkimutai**
The project involves creating the contact form using HTML and styling it using the CSS.
It also involves collaboration with the coding partners so as to modify the code.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -227,5 +234,28 @@ <h1>Skills</h1>
             </div>
         </div>
     </section>
+
+    <footer class="contact">
+        <div>
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <br>
+            <form>
```
Comment (aamirkhan2478) [inline index.html]: - [ ] Form uses the `action` and `method` attributes to integrate with [formspree service](https://formspree.io/html).
- [ ] Kindly add `method` and also use `action` attributes.
- [ ] `action` value should be similar to `https://formspree.io/f/{form_id}`
- [ ] `method` value equals `POST`

**Check below my suggestions:**
```suggestion
            <form method='POST' action=' https://formspree.io/f/{form_id}'>
```

- [ ] Please go to [formspree service](https://formspree.io/html) and follow the instruction. You will get one `form_id` from this website and then you just change this `id` with `{form_id}` like that `https://formspree.io/f/mzbolesd`.

File: index.html [hunk 2]
```diff
@@ -227,5 +234,28 @@ <h1>Skills</h1>
...
+                <input type="text" maxlength="30" placeholder="Full name" required>
+                <input type="email" placeholder="Email adress" required>
+                <textarea maxlength="500" placeholder="Enter text here" required></textarea>
+                <button name="submit">Get in touch</button>
+                <ul>
+                    <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href=""><i class="fa-brands fa-linkedin-in"></i></a></li>
+                    <li><a href=""><i class="fa-brands fa-github"></i></a></li>
+                    <li><a href=""><i class="fa-solid fa-envelope"></i></a></li>
+                </ul>
```
Comment (aamirkhan2478) [inline index.html]: - [ ] Kindly use the same social media icons as mentioned in the Figma design check below :point_down: 

Your Design:
Screenshot from 2023-02-21 21-01-12

Figma Design:
Screenshot from 2023-02-21 21-01-04

File: index.html [hunk 3]
```diff
@@ -227,5 +234,28 @@ <h1>Skills</h1>
...
+
+    <footer class="contact">
+        <div>
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <br>
+            <form>
+                <input type="text" maxlength="30" placeholder="Full name" required>
+                <input type="email" placeholder="Email adress" required>
+                <textarea maxlength="500" placeholder="Enter text here" required></textarea>
+                <button name="submit">Get in touch</button>
```
Comment (aamirkhan2478) [inline index.html]: - [ ] Please use the same button effects as mentioned in Figma Design your effects are not the same as Figma design check below :point_down: 
Screenshot from 2023-02-21 21-02-33

- [ ] Kindly fix these styles on your end. :slightly_smiling_face:

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @yegonkimutai , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linter checked

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

[OPTIONAL] Please add some descriptive description about your project in your README file check below :point_down: 
Screenshot from 2023-02-21 20-50-14

[OPTIONAL] Please remove `Server` and `Database` from your README file and add `HTML/CSS` inside `Client` in the `Tech Stack` section check below :point_down: 
Screenshot from 2023-02-21 20-49-52


 
You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -227,5 +235,29 @@ <h1>Skills</h1>
...
+
+    <footer class="contact">
+        <div>
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <br>
+            <form action="https://formspree.io/f/xrgvqloo" method="POST">
+                <input type="text" name="tect" maxlength="30" placeholder="Full name" required>
+                <input type="email" name="email" placeholder="Email adress" required>
+                <textarea maxlength="500" name="message" placeholder="Enter text here" required></textarea>
+                <button type="submit" name="submit">Get in touch</button>
```
Comment (gemmen29) [inline index.html:247]: - [ ] Kindly follow the style for the `Get in touch` button as requested by the previous reviewer.
image
When the button is active button looks like this:
image
When you hover with a mouse inside it button looks like this:
image
When you press and hold the button looks like this:
image
### Hint:
- You can use [Pseudo classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes).

Comment (gemmen29) [inline index.html:247]: - [ ] [OPTIONAL] Kindly increase the padding for the inputs and the `Get in touch` button to match the Figma design.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @yegonkimutai, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Highlights💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (victorokoroji) [inline index.html:247]: - [ ] Good job trying to implement the hover state of the buttons in your application. KIndly note that your hover state should be this 👇🏽 
image

but instead, you were using this 👇🏽 
image

Kindly fix it 💯

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi  @yegonkimutai, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (amon-cofie) [inline index.html:247]: You have done a great job so far @yegonkimutai 💪 but perhaps you didn't understand what the previous reviewer meant. I think the three color options are in triplets and so for instance, if your enabled button is green then the other states of the button should be in the other shades of green as provided and if your enabled state is blue then the other states of the button should be in the other shades of blue as provided 😉. Currently you have green on your enabled state and blue on your hover state : 
image 
image

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @yegonkimutai , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights

No linters errors ✔️
Descriptive pull request ✔️
Descriptive commit messages ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — APPROVED** [review]
Hello @yegonkimutai  👋,

_This is an additional review as a result of the second opinion request._

Kudos for the great work done so far! :clap:

I am very happy to say that on my end I didn't met the aspect mentioned by the previous reviewer. :confetti_ball: 
The buttons interactions the other project requirements were correctly implemented.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take a few hours!

Keep rocking 🚀!

**Pull request merged**